beacon-chain/core/blocks/block_operations.go

```diff
@@ -541,6 +541,51 @@ func VerifyAttestation(beaconState *pb.BeaconState, att *pb.Attestation, verifyS
 	return nil
 }
 
+// ConvertToIndexed converts attestation to (almost) indexed-verifiable form.
+//
+// Spec pseudocode definition:
+//   def convert_to_indexed(state: BeaconState, attestation: Attestation) -> IndexedAttestation:
+//     """
+//     Convert ``attestation`` to (almost) indexed-verifiable form.
+//     """
+//     attesting_indices = get_attesting_indices(state, attestation.data, attestation.aggregation_bitfield)
+//     custody_bit_1_indices = get_attesting_indices(state, attestation.data, attestation.custody_bitfield)
+//     custody_bit_0_indices = [index for index in attesting_indices if index not in custody_bit_1_indices]
+//     return IndexedAttestation(
+//         custody_bit_0_indices=custody_bit_0_indices,
+//         custody_bit_1_indices=custody_bit_1_indices,
+//         data=attestation.data,
+//         signature=attestation.signature,
+//     )
+func ConvertToIndexed(state *pb.BeaconState, attestation *pb.Attestation) (*pb.IndexedAttestation, error) {
+	attI, err := helpers.AttestationParticipants(state, attestation.Data, attestation.GetAggregationBitfield())
```

Comment:
why are you calling the getter? `attestation.GetAggregationBitfield()`

Reply:
changed